packages/sanity/src/core/version.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * TODO: rename to `VERSION`
```

Comment:
What is preventing us from acting on the todo?

Reply:
I don't know why this was introduced, but sure - will do the change and see what happens :-)

Reply:
Actually, will keep that out of this PR 😅